controllers/rhmi/rhmi_controller.go

```diff
@@ -1048,7 +1049,7 @@ func (r *RHMIReconciler) processStage(installation *rhmiv1alpha1.RHMI, stage *St
 			return rhmiv1alpha1.PhaseFailed, fmt.Errorf("failed to read productStatus config for %s: %v", string(productStatus.Name), err)
 		}
 
-		if productStatus.Phase == rhmiv1alpha1.PhaseCompleted {
+		if productStatus.Phase == rhmiv1alpha1.PhaseCompleted && productName != integreatlyv1alpha1.ProductGrafana {
```

Comment:
not required. GetWatchableCRDs for grafana returns `nil` so it doesn't need to be skipped. If you want to leave it in remove the GetWatchableCRDs from grafana recon

Reply:
done. Thank you